test/unit/product/product-component.js

```diff
@@ -1187,6 +1187,14 @@ describe('Product Component class', () => {
           assert.equal(image.srcLarge.maxHeight, image.srcLarge.maxWidth * 1.5);
         });
 
+        it('outputs the images alt text value when alt text is passed', () => {
```

Comment:
Could we update this test name to `returns the passed in image alt text if it is valid` for consistency with other test names, and to reflect how the function is used?